app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ActiveRecord::Base
+	belongs_to :user
+	belongs_to :postquote
+	validates :content, presence: true, uniqueness: true
+	validates :user_id, presence: true
```

Comment:
Try `validates :user` rather than `validates :user_id` - that way ActiveRecord will check that the user that the user id refers to in fact exists as well as that the field is not null.